docs/quick-start-guide.md

```diff
@@ -155,6 +158,14 @@ $ helm upgrade <YOUR-HELM-RELEASE-NAME> --set operatorImageName=org/image --set
 
 Refer to the Helm [documentation](https://docs.helm.sh/helm/#helm-upgrade) for more details on `helm upgrade`.
 
+## About the Spark Job Namespace
+
+The default SparkJobNamespace is `""`, as defined in the Helm chart's [README](https://github.com/helm/charts/blob/master/incubator/sparkoperator/README.md). Note that in the [Kubernetes apimachinery](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/apimachinery) project, the constants `NamespaceAll` and `NamespaceNone` are both defined as the empty string. In this case, the empty string represents `NamespaceAll`. When set to `""`, the Spark Operator supports deploying `SparkApplications` to all namespaces. The Helm chart will create a service account in the namespace where the spark-operator is deployed, but Helm skips setting up the RBAC used to give permission to the service account. In order to deploy jobs, you will need to ensure the driver pod's service account meets the criteria described in the [service accounts for driver pods](#about-the-service-account-for-driver-pods) section.
```

Comment:
Two things:
1. `SparkJobNamespace` -> `Spark Job Namespace` to be consistent. Then let's add a note after `Spark Job Namespace` that mentions the `sparkJobNamespace` parameter defined in the Helm chart (with ` quotes).
2. `but Helm skips setting up the RBAC used to give permission to the service account`. I would rephrase this as: `but Helm skips setting up the RBAC for driver pods of your Spark applications to be able to manipulate executor pods`.

Reply:
I added a commit to address both points.